lib/modelHelpers.py

```diff
@@ -32,40 +32,20 @@ def new_thread():
     actions.app.notify("Created a new thread")
 
 
-def push_context(context: str):
+def push_context(context: dict[str, any]):
     """Add the selected text to the stored context"""
     global stored_context
```

Comment:
Good with me in this PR, just as a comment I think we will eventually refactor this out into the GPTState class but no rush on this.